AUTHORS.rst

```diff
@@ -1,10 +1,15 @@
 =======
 Authors
 =======
+``21CMMC`` was originally written by `Brad Greig <https://github.com/BradGreig>`_.
+The updated interface that uses the Python-wrapped ``21cmFAST`` was primarily
+developed by `Steven Murray <https://github.com/steven-murray>`_.
+It is maintained by the full ``21cmFAST`` `collaboration <https://github.com/21cmFAST/people>`_.
 
-* Brad Greig - github.com/BradGreig
-* Steven Murray - github.com/steven-murray
+Here follows a list of individuals who have contributed in any way to ``21CMMC``:
 
-
-Contributors
-============
+* `Brad Greig <https://github.com/BradGreig>`_
+* `Steven Murray <https://github.com/steven-murray>`_
+* `Andrei Mesinger <https://github.com/andreimesinger>`_
+* `Catherine Watkinson <https://github.com/catherinewatkinson>`_
+* `Yuxiang Qin <https://github.com/qyx268>`_
```

Comment:
You should probably add Jaehong (he has contributed things to 21CMMC). Probably should add Tom as well

Reply:
Thanks, will add them in.